addOns/pscanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -83,5 +83,13 @@ <H2>Sub Resource Integrity Attribute Missing</H2>
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/SubResourceIntegrityAttributeScanner.java">SubResourceIntegrityAttributeScanner.java</a>
 
+<H2>Dangerous JS Functions</H2>
+This scanner checks for any dangerous JS functions present in a site response.<br>
+For more details see: https://github.com/zaproxy/zaproxy/issues/5673
```

Comment:
We should provide the details here instead (if needed), in other alert descriptions we don't link to the issues.